include/fuse_lowlevel.h

```diff
@@ -990,6 +990,11 @@ struct fuse_lowlevel_ops {
 	void (*bmap) (fuse_req_t req, fuse_ino_t ino, size_t blocksize,
 		      uint64_t idx);
 
+#if FUSE_USE_VERSION < 35
```

Comment:
I've only updated the non-lowlevel version in https://github.com/libfuse/libfuse/pull/600 but I think lowlevel version should be updated or the doc is pointing to the old signature.